packages/link/src/__demo__/examples/link.js

```diff
@@ -15,16 +15,41 @@
  */
 
 /* @flow */
+import { createStyledComponent } from '@mineral-ui/component-utils';
 import Link from '../../Link';
 
+const Big = createStyledComponent('p', {
+  fontSize: '1.5em'
+});
+const Bold = createStyledComponent('p', {
+  fontWeight: 700
+});
+const Serif = createStyledComponent('p', {
+  fontFamily: 'serif'
+});
+const Small = createStyledComponent('p', {
+  fontSize: '0.75em'
+});
+
```

Comment:
I disagree. Both the example description and live preview make it more than clear what's going on (Links look like whatever they're placed in). The actual code used to demonstrate that isn't important.